src/app/features/notification/notification.service.ts

```diff
@@ -81,7 +138,7 @@ export class NotificationService {
    * First retrieves the FCM token and then calls the notification sending function.
    */
   async sendNotification(payload: Record<string, any> = {}) {
```

Comment:
Yes, we can trigger notifications directly from the app. To do this, we simply need to provide the Firebase access token, the user's FCM token, and the notification payload (such as the message title, body, etc.). This will trigger the notification for the specified user's FCM token.